R/readworldsteel.R

```diff
@@ -56,7 +56,8 @@ readworldsteel <- function(subtype = 'detailed') {
           read_ods(path = file_path, sheet = sheet, na = '...') %>%
             as_tibble() %>%
             mutate(name = sheet) %>%
-            pivot_longer(c(-'country', -'name'), names_to = 'year', names_transform = list(year = as.integer))
+            pivot_longer(c(-'country', -'name'), names_to = 'year') %>%
+            mutate(!!sym("year") := as.integer(gsub("^X", "", !!sym("year"))))
```

Comment:
Are you not too young to have done a lot of Pascal programming? :)

Reply:
Happy to comply